text-ui-test/input.txt

```diff
@@ -0,0 +1,23 @@
+todo
```

Comment:
I like how your test is very detailed also!

Reply:
Agreed with the above comment! Very extensive testing✅